### PR TITLE
Use questions json and run cron

### PR DESCRIPTION
**Pull request by leo-step**
- Read & display data directly from questions.json instead of overwriting src/data/index.js
- Run cron job every Sunday at 5pm PST to automatically update question metadata

### CODE REVIEW

**Comment by leo-step** [comment]
Next up is checking companies and using "updated" field in questions.json as value for tooltip.

File: .github/workflows/run-cron.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: run-cron
 
-on: workflow_dispatch
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```
Comment (seanprashad) [inline .github/workflows/run-cron.yml:5]: For future reference, `00:00` UTC is `8:00pm EDT`/`5:00pm PDT`:

Screenshot 2022-08-01 at 10 26 56 PM

See https://crontab.guru/#0_0_*_*_0

File: src/components/Table/index.js [hunk 2]
```diff
@@ -266,9 +266,7 @@ const Table = () => {
             accessor: 'solutions',
             disableSortBy: true,
             Cell: cellInfo => {
-              const url = cellInfo.row.original.premium
-                ? `${cellInfo.row.original.url}/`
-                : cellInfo.row.original.url;
+              const url = `https://leetcode.com/problems/${cellInfo.row.original.url}/`;
```
Comment (seanprashad) [inline src/components/Table/index.js:269]: Thanks for the update here - I'm trying to remember why I had this conditional in the first place.. so weird 😅

**Review by seanprashad — APPROVED** [review]
Nice work @leo-step - you're on a roll!! 👏🏽

**Comment by seanprashad** [comment]
> Next up is checking companies and using "updated" field in questions.json as value for tooltip.

Makes sense to me! In order to grab the list of companies, we'll need to authenticate via the regular `/login` API path, retrieve the session token, and pass that in the payload with the GraphQL queries. I might be wrong, but that's how I'm thinking we should tackle it - we can leverage other great repos like https://github.com/realvect0r/leetcode-api/blob/32b5658fd8e5c27ebddfe7f1198218e9e13be64e/src/lib/leetcode.ts#L35 if that's easier

**Pull request merged**